ui_tests/pages/groups_page.py

```diff
@@ -0,0 +1,282 @@
+import time
+
+import pages.helper as helper
+from logger.logger import Logger
+from selenium.webdriver.common.by import By
+from webdriver.webriver_factory import WebDriverFactory
+
+logger = Logger().get_logger()
+driver = WebDriverFactory.get_driver()
+
+
+class GroupsPage:
+    def checkIfGroupsTableIsDisplayed(self):
```

Comment:
can we change the name to more pythonic syntax?

Reply:
changed